src/main/lua/exasolrls/rls_adapter.lua

```diff
@@ -61,13 +59,16 @@ end
 -- @return response containing the metadata for the virtual schema like table and column structure
 --
 function M.refresh(_, request)
-    local properties = request.schemaMetadataInfo.properties or {}
-    validate(properties)
-    local schema_metadata = metadata_reader.read(properties.SCHEMA_NAME)
-    return {type = "refresh", schemaMetadata = schema_metadata}
+    return {type = "refresh", schemaMetadata = handle_schema_scanning_request(request)}
 end
 
+---
+-- Alter the schema properties
+--
 function M.set_properties()
+    -- Not implemented yet:
+    -- https://github.com/exasol/row-level-security-lua/issues/89
+    return {type ="setProperties"}
```

Comment:
Shouldn't this throw an exception when it is not implemented?
```suggestion
    -- Not implemented yet:
    -- https://github.com/exasol/row-level-security-lua/issues/89
    return {type = "setProperties"}
```

Reply:
I will fix this one right after merging, before the release, so no need.